### DISCUSSION TITLE
[Bug]: Removing OpenAI from Endpoints causes error message in rag_api

### DISCUSSION THREAD

**Discussion by jowin202** [danny-avila/LibreChat#3272] [Troubleshooting] [answered]
### What happened?

Hello,
I would like to remove all Endpoints in Chat except my local Ollama. Therefore, I removed all keys and api tokens (assistants, azure, bing, google, ...) from the .env file. This works pretty well except for OPENAI_API_KEY=
Whenever i comment this out, I receive an error in rag_api container 

>   Did not find openai_api_key, please add an environment variable `OPENAI_API_KEY` which contains it, or pass `openai_api_key` as a named parameter. (type=value_error)

but otherwise, if I keep OPENAI_API_KEY=user_provided, I still have OpenAI in the list of endpoints in the gui (where it shouldn't be). is there a way to remove openai without getting rag_api crashed?

### Steps to Reproduce

1. pull from repo
2. try to remove openai_api_key 
3. error

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
Did not find openai_api_key, please add an environment variable `OPENAI_API_KEY` which contains it, or pass `openai_api_key` as a named parameter. (type=value_error)
Traceback (most recent call last):
  File "/app/main.py", line 46, in <module>
    from psql import PSQLDatabase, ensure_custom_id_index_on_embedding, pg_health_check
  File "/app/psql.py", line 3, in <module>
    from config import DSN, logger
  File "/app/config.py", line 226, in <module>
    embeddings = init_embeddings(EMBEDDINGS_PROVIDER, EMBEDDINGS_MODEL)
  File "/app/config.py", line 180, in init_embeddings
    return OpenAIEmbeddings(
  File "/usr/local/lib/python3.10/site-packages/pydantic/v1/main.py", line 341, in __init__
    raise validation_error
pydantic.v1.error_wrappers.ValidationError: 1 validation error for OpenAIEmbeddings
__root__
  Did not find openai_api_key, please add an environment variable `OPENAI_API_KEY` which contains it, or pass `openai_api_key` as a named parameter. (type=value_error)
```

### Screenshots

![openai_error](https://github.com/danny-avila/LibreChat/assets/35803880/99e6549a-333a-427c-ae96-d09b8260c7e8)

openai and plugin dropdown should be removed.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Use `RAG_OPENAI_API_KEY` instead, unless you are using ollama or something else for RAG

https://www.librechat.ai/docs/configuration/rag_api#docker-setup---quick-start

**Reply by danny-avila** [reply, marked as answer]
also in your .env file, you should have this:

`ENDPOINTS=custom`

**Reply by jowin202** [reply]
thank you
thank you
thank you

this solves the problem
